lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/aten_ltc_ts_type.cpp

```diff
@@ -635,20 +635,6 @@ at::Tensor LazyNativeFunctions::max_pool3d(
       self, kernel_size, stride, padding, dilation, ceil_mode);
 }
 
-at::Tensor LazyNativeFunctions::mm(const at::Tensor& self,
-                                   const at::Tensor& mat2) {
-  LTC_FN_COUNTER("lazy::");
```

Comment:
ahh i was wondering why the 'counter' disappeared from the metrics output diff you pasted.  I guess I missed putting a counter into the codegenned ops.

Reply:
Added this macro call into the code gen thingy.